scalafix/input/src/main/scala/fix/ZIOSpec.scala

```diff
@@ -1638,19 +1638,19 @@ object ZIOSpec extends DefaultRunnableSpec {
         } yield assert(res._1)(equalTo(List(0, 2, 4, 6, 8))) && assert(res._2)(equalTo(List(1, 3, 5, 7, 9)))
       }
     ),
-    suite("provideCustomServices")(
+    suite("provideCustomLayer")(
       testM("provides the part of the environment that is not part of the `ZEnv`") {
-        val loggingServiceBuilder: ZServiceBuilder[Any, Nothing, Logging] = Logging.live
```

Comment:
Our `input` files should still have the old types, since they compile with the old version of ZIO

Reply:
I may have been a little aggressive in excising any references to `ZLayer`. 😅